app/src/main/java/com/example/eric/socials/DetailActivity.java

```diff
@@ -0,0 +1,181 @@
+package com.example.eric.socials;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Display;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.storage.FirebaseStorage;
+
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+
+public class DetailActivity extends AppCompatActivity implements View.OnClickListener{
+
+    TextView socialName;
+    TextView emailAddress;
+    TextView socialDescription;
+
+    ImageView socialImage;
+
+    Button interestedButton;
+    Button numRSVPButton;
+
+    DatabaseReference myRef;
+
+    private static FirebaseAuth mAuth;
+    private static FirebaseUser mUser;
+
+    Intent grabIntent;
+
+    ArrayList<String> usersInterested;
+
+    boolean isInterested = false;
+    int intNumRSVP;
+
+    @Override
+    public void onClick(View v){            // Handles Cases for clicking Interested & Num RSVP Buttons
+        switch (v.getId()) {
+            case R.id.numRSVPButton:
```

Comment:
Code in each case in this switch statement could be modularized into functions to make it easier to read. Like you could have numRSVPButtonClicked(), interestedButtonClicked(), etc